.github/workflows/update-gh-pages-on-release.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         git worktree add ../docs gh-pages
     - name: Update the docs on the gh-pages branch
       run: |
-        python docs/update_gh_pages.py docs/build/html ../docs --tag ${{ GITHUB_REF_NAME }}
+        python docs/update_gh_pages.py docs/build/html ../docs --tag ${{ github.ref_name }}
```

Comment:
This was a really dumb mistake. I think `${{ env.GITHUB_REF_NAME }}` would also work here, but this seems to work in testing.